### ISSUE TITLE
Adding base_checksum to bat so it will work

### ISSUE THREAD

**Issue by Domenik2105** [Vaweila/Aging-Knights#1]
https://steamcommunity.com/app/736820/discussions/0/5704402939706321181/

Without the base_checksum.txt you can not selecte the mod in the menu

**Comment by kevincompton** [comment]
this is probably my issue, i'm unclear what should be in the checksum file

**Comment by kevincompton** [comment]
yep that was it, it generates a unique key per update I assume? So would need to be maintained in the top level folder
